app/views/certificate/VatCertificate.scala.html

```diff
@@ -81,6 +81,16 @@
   List("1", "Z1").contains(model.businessTypeMsgKey.split('.').last)
 }
 
+    @webchatLink = {
+        <br>
+        <br>
+        <p class="govuk-body">
+            <a class="govuk-link" href="https://www.tax.service.gov.uk/ask-hmrc/chat/vat-online?ds" target="_blank" id="webchatLink-id">@messages("webchat.askHMRC")</a>
```

Comment:
Can you make it so the link doesn't hand off to the production url locally and in the test environments please. i.e. create the handoff in app config and use the services host and port to get the base url